index.html

```diff
@@ -0,0 +1,49 @@
+        </div>
+        <div class="socials">
+            <div class="twitter">
+                <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M19.7665 0.369765C19.4331 -0.0289051 18.8665 -0.128573 18.4331 0.17043L16.4998 1.49933C15.5998 0.535877 14.3331 0.00431735 12.9665 0.00431735C10.2665 0.00431735 8.03313 2.197 8.03313 4.92125C8.03313 5.31992 8.06646 5.68537 8.16646 6.05081C5.83313 5.55248 4.43313 3.85813 3.8998 0.801657C3.83313 0.402987 3.53313 0.103985 3.13313 0.00431735C2.76646 -0.0953502 2.36646 0.0707624 2.13313 0.369765C2.06646 0.43621 -3.43354 8.07739 4.53313 14.5558C3.76646 16.1837 1.46646 17.579 0.566463 18.0441C0.13313 18.2435 -0.06687 18.7086 0.03313 19.1737C0.13313 19.6388 0.53313 19.9378 0.999797 19.9378H2.73313C12.8665 19.9378 17.4665 14.9212 17.5665 3.69202L19.6665 1.69867C20.0665 1.36644 20.0998 0.768435 19.7665 0.369765Z" fill="#DBD8D7"/>
+                </svg>
+                </div>
+                <div class="github">
+                    <svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M17.9989 7.46114C17.9989 6.32124 17.6329 5.25043 16.9342 4.31779C17.2004 3.41969 17.2669 2.27979 17.2004 0.967185C17.1671 0.414508 16.7345 0 16.2022 0C15.9028 0 13.3741 0.0345423 11.9101 1.38169C10.6458 1.1399 9.31495 1.1399 8.01736 1.38169C6.58667 0.0345423 4.05802 0 3.7253 0C3.19296 0 2.76042 0.414508 2.72715 0.967185C2.62734 2.27979 2.72715 3.41969 2.99333 4.31779C2.29462 5.28497 1.92863 6.35579 1.92863 7.46114C1.92863 9.8791 3.7253 12.0553 6.45358 13.0915C6.35377 13.2988 6.28723 13.5406 6.22068 13.7824C3.32604 13.4715 1.9619 10.7427 1.89536 10.639C1.66246 10.1209 1.06357 9.91364 0.56449 10.19C0.0654144 10.4318 -0.134216 11.0535 0.131958 11.5717C0.198501 11.7444 2.02845 15.4404 6.05432 15.8549V18.9637C6.05432 19.5509 6.48686 20 7.05248 20H12.875C13.4406 20 13.8732 19.5509 13.8732 18.9637V14.8532C13.8732 14.2314 13.7401 13.6442 13.5072 13.1261C16.2022 12.0553 17.9989 9.91364 17.9989 7.46114Z" fill="#DBD8D7"/>
+                    </svg>
+                    </div>
+                    <div class="medium">
+                     
+<svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" fill="#DBD8D7"/>
+    </svg>
+       
+                    </div>
+        </div>
```

Comment:
- [ ] For  list of Social  icons kindly  uses a `ul` tag to group them, and `li` per icon; each icon has a link `a` tag to let you connect your social networks with the icons, using `href.`
- [ ] For Social icons please try to follow the Figma design Requirement .You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
For Social icons your project look likes this : 
![try](https://user-images.githubusercontent.com/93598189/185462913-03b2d14c-2abf-477f-9d15-9ebf55b03d9d.PNG)

But Figma design Look like this : 

![fig](https://user-images.githubusercontent.com/93598189/185463345-06c6d033-2718-4366-a4b3-2acaf98295db.PNG)